swarm/src/lib.rs

```diff
@@ -1508,24 +1527,79 @@ impl<TBehaviour> SwarmBuilder<TBehaviour>
 where
     TBehaviour: NetworkBehaviour,
 {
-    /// Creates a new [`SwarmBuilder`] from the given transport, behaviour, local peer ID and
-    /// executor. The `Swarm` with its underlying `Network` is obtained via
-    /// [`SwarmBuilder::build`].
-    pub fn with_executor(
```

Comment:
This diff makes it look like this function was removed when in fact it is now on line 1594. Can we restore the order please?